Extras/README.md

```diff
@@ -55,62 +55,56 @@ wealth of information.
 
 [Relay Extras](https://youtu.be/qXykEckzEgs)
 
-## ListMarkets <exchange> <ratelimit>
```

Comment:
Something about however markdown gets rendered was hiding these arguments from being displayed, so I copied them on to a separate line